install-dev/data/xml/configuration.xml

```diff
@@ -228,7 +228,7 @@
     <value>1</value>
   </configuration>
   <configuration id="PS_ATTRIBUTE_CATEGORY_DISPLAY" name="PS_ATTRIBUTE_CATEGORY_DISPLAY">
-    <value>1</value>
+    <value>0</value>
   </configuration>
```

Comment:
allow_add_variant_to_cart_from_listing (PS_ATTRIBUTE_CATEGORY_DISPLAY) should be disabled by default. If I understand correctly the default values are taken from this xml.
https://github.com/PrestaShop/PrestaShop/pull/22471#issuecomment-748831978

Reply:
OK with this @PrestaShop/product-team ?

Reply:
Yep, thanks for asking, @Progi1984! As written in the Product Settings page [specifications](https://github.com/PrestaShop/prestashop-specs/blob/master/back-office/shop-parameters/product-settings/product-settings-page.md#product-page):

> Display the "Add to cart" button when a product has attributes. **Important note: this feature does not work with the Classic default theme.** It has been removed from the front office in 1.7.x because many users added products to their cart without selecting a combination. [...] **It is still in the back office since it might work with other themes. By default, it should be disabled.** [...]

Reply:
Ok, so I approve.